test/unit/gateways/mercado_pago_test.rb

```diff
@@ -80,6 +81,16 @@ def test_successful_purchase_with_naranja
     assert response.test?
   end
 
+  # def test_successful_purchase_with_notification_url
```

Comment:
It seems like the new unit test for this new field shouldn't be commented out?

Also, because this is a unit test for a GSF, you will likely need to use the `stub_comms` method to test what is being sent in the request instead of what's coming back in the response since GSFs are often not sent back in the response from the gateway.

See the [other unit test examples](https://github.com/activemerchant/active_merchant/blob/master/test/unit/gateways/mercado_pago_test.rb#L288).

Reply:
Oops that was definitely not intentional